packages/ui-extensions-dev-console/src/sections/Extensions/Extensions.tsx

```diff
@@ -34,7 +37,14 @@ export function Extensions() {
             <th>{i18n.translate('extensionList.name')}</th>
             <th>{i18n.translate('extensionList.preview')}</th>
             <th>{i18n.translate('extensionList.mobile')}</th>
-            <th>{i18n.translate('extensionList.view')}</th>
+            <th>
+              <span className={styles.WithIcon}>
+                {i18n.translate('extensionList.view')}
+                <Tooltip text={i18n.translate('tooltips.viewColumnHeader')}>
+                  <Icon source={QuestionMarkMajor} />
```

Comment:
I think the color of the Icon doesn't match the deigns, please could we update it?